docs/03-endpoints/api-util/health.md

```diff
@@ -0,0 +1,23 @@
+<!---
+ * Copyright © 2021 - 2022 Swiss National Data and Service Center for the Humanities and/or DaSCH Service Platform contributors.
+ * SPDX-License-Identifier: Apache-2.0
+-->
+
+# Health
+
+The health endpoint provides information about the health state of the dsp-stack.
+
+
+## Example request
+`GET /health`
+
+
+## Example response
+```
```

Comment:
again, I'd prefer empty lines after headers

Reply:
also, I'd mark it as `json` so that one gets syntax highlighting

Reply:
(I can't do the suggest because you can't have triple backtick in suggestions... sorry)